src/patternfly/components/Button/styles.scss

```diff
@@ -81,32 +81,32 @@
 
   // Tertiary btn
-  --pf-c-button--m-tertiary--BackgroundColor: transparent;
-  --pf-c-button--m-tertiary--BorderColor: var(--pf-global--secondary-color);
-  --pf-c-button--m-tertiary--Color: var(--pf-color-black-900);
-  --pf-c-button--m-tertiary--hover--BackgroundColor: transparent;
-  --pf-c-button--m-tertiary--hover--BorderColor: var(--pf-global--secondary-color);
-  --pf-c-button--m-tertiary--hover--Color: var(--pf-color-black-900);
-  --pf-c-button--m-tertiary--focus--BackgroundColor: transparent;
-  --pf-c-button--m-tertiary--focus--BorderColor: var(--pf-global--secondary-color);
-  --pf-c-button--m-tertiary--focus--Color: var(--pf-color-black-900);
-  --pf-c-button--m-tertiary--active--BackgroundColor: transparent;
-  --pf-c-button--m-tertiary--active--BorderColor: var(--pf-global--secondary-color);
-  --pf-c-button--m-tertiary--active--Color: var(--pf-color-black-900);
+  --pf-c-button--is-tertiary--BackgroundColor: transparent;
+  --pf-c-button--is-tertiary--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-button--is-tertiary--Color: var(--pf-color-black-900);
+  --pf-c-button--is-tertiary--hover--BackgroundColor: transparent;
+  --pf-c-button--is-tertiary--hover--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-button--is-tertiary--hover--Color: var(--pf-color-black-900);
+  --pf-c-button--is-tertiary--focus--BackgroundColor: transparent;
+  --pf-c-button--is-tertiary--focus--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-button--is-tertiary--focus--Color: var(--pf-color-black-900);
+  --pf-c-button--is-tertiary--active--BackgroundColor: transparent;
+  --pf-c-button--is-tertiary--active--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-button--is-tertiary--active--Color: var(--pf-color-black-900);
 
   // danger btn
-  --pf-c-button--m-danger--BackgroundColor: var(--pf-global--danger-color);
-  --pf-c-button--m-danger--BorderColor: var(--pf-global--danger-color);
-  --pf-c-button--m-danger--Color: var(--pf-global--Color--light-100);
-  --pf-c-button--m-danger--hover--BackgroundColor: var(--pf-global--danger-color--dark);
-  --pf-c-button--m-danger--hover--BorderColor: var(--pf-c-button--m-danger--hover--BackgroundColor);
-  --pf-c-button--m-danger--hover--Color: var(--pf-global--Color--light-100);
-  --pf-c-button--m-danger--focus--BackgroundColor: var(--pf-global--danger-color--dark);
-  --pf-c-button--m-danger--focus--BorderColor: var(--pf-c-button--m-danger--hover--BackgroundColor);
-  --pf-c-button--m-danger--focus--Color: var(--pf-global--Color--light-100);
-  --pf-c-button--m-danger--active--BackgroundColor: var(--pf-global--danger-color--dark);
-  --pf-c-button--m-danger--active--BorderColor: var(--pf-c-button--m-danger--hover--BackgroundColor);
-  --pf-c-button--m-danger--active--Color: var(--pf-global--Color--light-100);
+  --pf-c-button--is-danger--BackgroundColor: var(--pf-global--danger-color--100);
```

Comment:
Looks like maybe a bad merge here - can you do a quick replace from -is- to -m- in this file?

Reply:
updated :)